.github/BOTMETA.yml

```diff
@@ -229,6 +229,7 @@ files:
   $modules/cloud/docker/docker_network.py: keitwb olsaki
   $modules/cloud/docker/docker_secret.py:
   $modules/cloud/docker/docker_service.py: $team_ansible
+  $modules/cloud/docker/docker_swarm.py: tbouvet
```

Comment:
This is not needed, and even unwanted. If you are listed as the author, you will automatically be considered like this. So we use the BOTMETA.yml as an addendum to what the module already has.

So please remove this.

Reply:
I have cleaned up the windows and vmware sections, and while they have a massive amount of modules, the actual statements in BOTMETA.yml are minimal. And that's how we want it.

Reply:
Ok I didn't know. I solve it.